kernelci/config/__init__.py

```diff
@@ -78,6 +78,46 @@ def load_yaml(config_path, validate_entries=None):
     return config
 
 
+def _merge_trees(a, b):
+    """Merge two values loaded from YAML"""
+    if isinstance(a, dict) and isinstance(b, dict):
+        all_keys = set(a.keys())
+        all_keys.update(b.keys())
+        all_keys = list(all_keys)
+        res = dict()
+        for k in all_keys:
+            if (k in a) and (k in b):
+                res[k] = _merge_trees(a[k], b[k])
+            elif k in a:
+                res[k] = a[k]
+            else:
+                res[k] = b[k]
+        return res
+    elif isinstance(a, list) and isinstance(b, list):
+        return a + b
+    else:
+        return b
+
+
+def load_all_yaml(config_paths, validate_entries=None):
+    """Load the YAML configuration
+
+    Load all the YAML files in all the specific configuration
+    directories and aggregate them together. Later directories take
+    precedence over earlier ones. This enables combining sources of
+    configuration data from multiple places.
+
+    *config_path* is an ordered list of YAML configuration directories
+                  or YAML files, with later entries having higher
+                  priority
```

Comment:
I think it is `config_paths` now. 

Can you also add a description of `validate_entries`

Reply:
I don't believe `validate_entries` is currently used; I've noted that the value is passed unchanged to the underlying load function, and I've added a note to that function that it doesn't currently consume that flag.